python/tests/wrapper/test_02_regularizers.py

```diff
@@ -0,0 +1,133 @@
+        7: 1564.405
+    }
+    sparsity_tol = 0.001
+    expected_phi_sparsity_value_on_iteration = {
+        0: 0.060,
+        1: 0.121,
+        2: 0.210,
+        3: 0.312,
+        4: 0.393,
+        5: 0.452,
+        6: 0.495,
+        7: 0.528
+    }
+    expected_theta_sparsity_value_on_iteration = {
+        0: 0.012,
+        1: 0.035,
+        2: 0.143,
+        3: 0.245,
+        4: 0.314,
+        5: 0.335,
+        6: 0.345,
+        7: 0.349
+    }
+
+    batches_folder = tempfile.mkdtemp()
+    try:
+        # Create the instance of low-level API and helper object
+        lib = artm.wrapper.LibArtm()
+        helper = helpers.TestHelper(lib)
+        
+        # Parse collection from disk
+        helper.parse_collection_uci(os.path.join(os.getcwd(), docword),
+                                    os.path.join(os.getcwd(), vocab),
+                                    batches_folder,
+                                    dictionary_name)
+
+        # Create master component and scores
+        scores = [('Perplexity', constants.ScoreConfig_Type_Perplexity, messages.PerplexityScoreConfig()),
+                  ('SparsityPhi', constants.ScoreConfig_Type_SparsityPhi, messages.SparsityPhiScoreConfig()),
+                  ('SparsityTheta', constants.ScoreConfig_Type_SparsityTheta, messages.SparsityThetaScoreConfig()),
+                  ('TopTokens', constants.ScoreConfig_Type_TopTokens, messages.TopTokensScoreConfig())]
+        master_id = helper.create_master_component(scores=scores)
+
+        # Import the collection dictionary
+        helper.import_dictionary(master_id, os.path.join(batches_folder, dictionary_name), dictionary_name)
+
+        # Configure basic regularizers
+        helper.create_smooth_sparse_phi_regularizer(master_id, 'SmoothSparsePhi')
+        helper.create_smooth_sparse_theta_regularizer(master_id, 'SmoothSparseTheta')
+        helper.create_decorrelator_phi_regularizer(master_id, 'DecorrelatorPhi')
+
+        # Initialize model
+        helper.initialize_model(master_id, pwt, num_topics, source_type='dictionary', dictionary_name=dictionary_name)
+
+        for iter in xrange(num_outer_iterations):
+            # Invoke one scan of the collection, regularize and normalize Phi
+            helper.process_batches(master_id, pwt, nwt, num_inner_iterations,
+                                   batches_folder, ['SmoothSparseTheta'], [smsp_theta_tau])
+            helper.regularize_model(master_id, pwt, nwt, rwt,
+                                    ['SmoothSparsePhi', 'DecorrelatorPhi'], [smsp_phi_tau, decor_phi_tau])
+            helper.normalize_model(master_id, pwt, nwt, rwt)   
+
+            # Retrieve scores
+            perplexity_score = messages.PerplexityScore()
+            helper.retrieve_score(master_id, pwt, 'Perplexity', perplexity_score)
+
+            sparsity_phi_score = messages.SparsityPhiScore()
+            helper.retrieve_score(master_id, pwt, 'SparsityPhi', sparsity_phi_score)
```

Comment:
I would prefer this syntax:
'''
sparsity_phi_score = helper.retrieve_score(master_id, pwt, 'SparsityPhi)
'''
Are there any downsides to this solution?